saleor/graphql/account/schema.py

```diff
@@ -194,7 +194,7 @@ def resolve_permission_group(_root, _info, *, id):
     @staticmethod
     def resolve_me(_root, info):
         user = info.context.user
-        return user if user.is_authenticated else None
+        return user if bool(user) else None
```

Comment:
You don't need to cast it on `bool`.

Reply:
I need, as this is SimpleLazyObject, and otherwise, I will return a lazy object instead of a user instance.

Reply:
I see. Thanks for clarification.